otcextensions/osclient/cce/v2/cluster.py

```diff
@@ -97,18 +97,42 @@ def get_parser(self, prog_name):
             metavar='<cluster>',
             help=_('ID of the cluster')
         )
+        parser.add_argument(
+            '--wait',
+            action='store_true',
+            help=('Wait for the instance to become active')
+        )
+        parser.add_argument(
+            '--wait-interval',
+            type=int,
+            help=_('Interval for checking status')
+        )
+        parser.add_argument(
+            '--wait-timeout',
+            type=int,
+            help=_('Wait timeout')
+        )
         return parser
 
     def take_action(self, parsed_args):
 
+        attrs = {}
+        for attr in [
+            'cluster', 'wait', 'wait_timeout', 'wait_interval'
+        ]:
+            if getattr(parsed_args, attr):
+                attrs[attr] = getattr(parsed_args, attr)
+
+        if not parsed_args.wait:
+            attrs['wait'] = False
+
         if parsed_args.cluster:
-            client = self.app.client_manager.cce
-            cluster = client.find_cluster(parsed_args.cluster,
```

Comment:
You have deleted find_cluster. Please change: "ID of the cluster" to "Name or ID of the cluster." in line 98 as well to let the user know that name is also possible.

Reply:
done